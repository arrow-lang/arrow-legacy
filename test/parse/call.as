def main() {
  // Infinite loop here we come
  main();
}
