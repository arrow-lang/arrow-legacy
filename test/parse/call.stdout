<?xml version="1.0" encoding="utf-8"?>
<Module>
 <Function name="main">
  <Parameters/>
  <Sequence>
   <Call>
    <Expression>
     <Identifier>main</Identifier>
    </Expression>
    <Arguments/>
   </Call>
  </Sequence>
 </Function>
</Module>
